src/FlowDiagnostics.jl

```diff
@@ -255,12 +255,22 @@ end
 #----
 
 #+++++ Tracer variance dissipation
+@inline function _calc_κᶜᶜᶜ(i, j, k, grid, closure::SmagorinskyLilly, buoyancy, velocities, tracers, ::Val{tracer_index}) where {tracer_index}
+    νₑ = calc_νᶜᶜᶜ(i, j, k, grid, closure, buoyancy, velocities, tracers)
+    @inbounds Pr = closure.Pr[tracer_index]
+    return νₑ / Pr
+end
+
+@inline function _calc_κᶜᶜᶜ(i, j, k, grid, closure::AnisotropicMinimumDissipation, buoyancy, velocities, tracers, id::Val{tracer_index}) where {tracer_index}
+    return _calc_κᶜᶜᶜ(i, j, k, grid, closure, tracers, id, velocities)
+end
+
 @inline function isotropic_tracer_variance_dissipation_rate_ccc(i, j, k, grid, c, velocities, p)
     dcdx² = ℑxᶜᵃᵃ(i, j, k, grid, fψ², ∂xᶠᶜᶜ, c) # C, C, C  → F, C, C  → C, C, C
     dcdy² = ℑyᵃᶜᵃ(i, j, k, grid, fψ², ∂yᶜᶠᶜ, c) # C, C, C  → C, F, C  → C, C, C
     dcdz² = ℑzᵃᵃᶜ(i, j, k, grid, fψ², ∂zᶜᶜᶠ, c) # C, C, C  → C, C, F  → C, C, C
 
-    κ = _calc_κᶜᶜᶜ(i, j, k, grid, p.closure, c, p.id, velocities)
+    κ = _calc_κᶜᶜᶜ(i, j, k, grid, p.closure, p.buoyancy, velocities, tracers, p.id)
```

Comment:
I'm not sure this formula is the correct discrete dissipation rate formula, because `κᶜᶜᶜ` is never invoked by the solver.

Reply:
So far I haven't paid much attention to the conservative forms when implementing diagnostics since that sort of precision hasn't been important for the research I've done with these. But I suspect you're right!

Should we include a disclaimer in the docstrings for users until we implement the actual conservative version? (Asking for diagnostics in this package in general, not just this one.)

Reply:
I think it's just that using `κᶜᶜᶜ` to calculate the tracer variance is an approximation; it isn't the exact tracer dissipation. I think for LES this could be an issue, but for DNS it may not be.

The other issue is more mundane: its actually harder to write generic code for this approximation. If we use the exact tracer variance we write pretty generic code as expressed below.

I suspect this will be true across the board --- and therein will lie a real value for this package, because it can be challenging to find the staggered-grid discrete analogs of high-order quantities that we want to calculate. So it's good to figure that out once (here).